Lesson14/gökçe/script.js

```diff
@@ -0,0 +1,20 @@
+// 9. Declare a variable number6 and initialize it to 6. Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
+let number6= 6 ;
+let number7= ++number6;
```

Comment:
Since value of `number7` never changes, it's safer to use `const` keyword.
```suggestion
const number7= ++number6;
```